src/engraving/libmscore/masterscore.cpp

```diff
@@ -529,7 +529,7 @@ Score::FileError MasterScore::read(XmlReader& e, bool ignoreVersionError, mu::en
                 ++counter;
 
                 if (style().defaultStyleVersion() == 400) {
-                    int k =1;
+                    //int k =1;
```

Comment:
Oh oh (

Reply:
Well, commenting it out is really just band aid here ;-)

Reply:
It is all but 1 of 97 warnings, the other 96 stem from the `class` vs. `struct` thing